app/utils/promise-proxy-object.js

```diff
@@ -4,4 +4,4 @@ import PromiseProxyMixin from '@ember/object/promise-proxy-mixin';
 // see https://emberjs.com/api/ember/3.7/classes/PromiseProxyMixin for
 // info on how this is used.
 
-export default ObjectProxy.extend(PromiseProxyMixin);
+export default class PromiseProxyObject extends ObjectProxy.extend(PromiseProxyMixin) {}
```

Comment:
I think this util can be removed, the code that used this seems to be removed as well.

Reply:
should be ok to remove indeed :+1: